SimpleBlockX4.md

```diff
@@ -0,0 +1,131 @@
+
+#### Creation of Void/Vacuum Volume and Graveyard
+_The vacuum and graveyard are volumes that are not part of the model being analyzed, but play a role in the final analysis. A method is implemented automatically during a DAG-MCNP5 run that creates a void volume in all spaces without geometry. This means that while one can explicitly create void volumes, it is often not necessary. These volumes could also be created in another CAD program and then assigned a material in the exact same way, but this is an example of how to create these volumes in Trelis and Cubit._
+
+  1. Create a cube large enough to envelop all other volumes.
+      * If you already adjusted the scale of the objects they should be 100 units on every side with 50 units between them. A cube with side length of 500 works for these purposes
+      * In command panel select __Geometry -> Volume -> Create__ and select __Brick__ from the drop down menu. Type 500 in each dimension box __(width, height, depth)__ and press __Apply__ to create the block.
+      * For command line the general form is `create [shape] [dimensions based on shape]`. For a simple cube 500x500x500 you can type `create brick x 500`.
+  2. Create another slightly larger cube to be used for graveyard.
+      * The graveyard must be a volume, but the size does not matter, only that it fully surrounds the volumes. For this we will use a 750 unit brick.
+      * In command panel create another brick in the same way as part 1, using 750 as the new dimension in each dimension box.
+      * In command line type `create brick x 750`
+  3. Center volumes within the larger cubes.
+      * To better view where the volumes are within the larger cubes select the wireframe option in the top toolbar. It is directly to the right of the undo arrow on the toolbar.
+      * Most likely the volumes will be slightly off centered in the x direction, but the larger bricks will be centered with one another. This may vary based on the initial CAD program.
+      * To see the blocks better you can click on the axis on the bottom left of the volume viewing area to get a straight on (x,y) view.
+      * To move the blocks in command panel select __Geometry -> Volume -> Transform__ and select __Move__ from the drop down menu. Type `5 6` into the __Volume ID(s)__ box. Then __Select Method -> Distance__ and type 100 into the __X Distance__ box. This distance/direction may vary.
+  4. Hollow out large cubes to match functionality.
+      * In command panel select __Geometry -> Volume -> Boolean__ and select __Subtract__ from the dropdown menu. First select volume 6 as volume A and volume 5 as volume B. Check __Keep Originals__ and click __Apply__. You can then delete volume 6 by selecting __Geometry -> Volume -> Delete__ and typing 6 into the __Volume ID(s)__ box and clicking apply.
+      * In command line type `subtract vol 5 from 6 keep` Then type `delete vol 6` to get rid of the original large box.

```

Comment:
I believe just doing `subtract vol 5 from 6` automatically removes volume 6. If you present the method you have described, you should add a note about what `keep` is doing and why you then need to delete vol 6.